services/web/server/src/simcore_service_webserver/api_keys/_repository.py

```diff
@@ -21,6 +21,10 @@
 _logger = logging.getLogger(__name__)
 
 
+def _hash_secret(secret: str) -> sa.sql.ClauseElement:
+    return sa.func.crypt(secret, sa.func.gen_salt("bf", 10))
```

Comment:
so we have that ```bf``` string in at least 3 separate locations. do you have a test checking that it is everywhere the same?

Reply:
The operation is not reversible and the salt is included in the hash, why it should be the same everywhere?